package.json

```diff
@@ -65,7 +65,7 @@
 		"tslint": "^5.14.0",
 		"tslint-config-standard": "^8.0.1",
 		"tslint-react": "^4.0.0",
-		"typescript": "^3.4.1",
+		"typescript": "3.3.4000",
```

Comment:
downgrade typescript as there is a performance issue with TS 3.4.1 working with styled component